src/main/resources/google_checks.xml

```diff
@@ -288,11 +288,13 @@
                     METHOD_DEF, QUESTION, RESOURCE_SPECIFICATION, SUPER_CTOR_CALL, LAMBDA,
                     RECORD_DEF"/>
     </module>
+    <!-- TYPE_EXTENSION_AND token for OperatorWrap explained in https://github.com/checkstyle/checkstyle/pull/9386 -->
```

Comment:
I think it is more valuable to explain why we are not including `COLON`, than to explain why we added `TYPE_EXTENSION_AND`. We have git blame to see issue/PR where code was added to find out why; but if code doesn't exist, it is harder to find out why it is not there. Do you agree?

Reply:
Yeah, will change it to ` <!-- COLON token for OperatorWrap explained in https://github.com/checkstyle/checkstyle/pull/9386 -->`

Reply:
Instead of `<!-- COLON token for OperatorWrap explained in https://github.com/checkstyle/checkstyle/pull/9386 -->`
perhaps do `<!-- Absence of COLON token for OperatorWrap explained in https://github.com/checkstyle/checkstyle/pull/9386 -->` ?

Reply:
Ohkay, that seems more apt, will change.

Reply:
@bossever I just had a thought; we should link to the issue, instead of the PR, since that is what is included in release notes. This way will be more consistent.  Also, please post your research and explanation of tokens in PR description to the issue so that all information can be found there. I should have considered this earlier.

Reply:
Linking the issue seems logical, I think we should do that, As for the explanation, should I just link the comment in the issue, or repost the contents in issue thread?

Reply:
Just repost the entire contents of this PR description into the issue thread.

Reply:
Resolved.